variables.tf

```diff
@@ -41,61 +47,26 @@ variable "prevent_destroy" {
   default     = true
 }
 
-variable "set_owners_for" {
-  description = "Name of keys for which owners will be set."
-  type        = list(string)
-  default     = []
-}
-
-variable "owners" {
-  description = "List of comma-separated owners for each key declared in set_owners_for."
-  type        = list(string)
-  default     = []
-}
-
-variable "set_encrypters_for" {
-  description = "Name of keys for which encrypters will be set."
-  type        = list(string)
-  default     = []
-}
-
-variable "encrypters" {
-  description = "List of comma-separated owners for each key declared in set_encrypters_for."
-  type        = list(string)
+variable "acl" {
+  description = "Access control list for the managed keys."
+  type        = list(any)
   default     = []
 }
 
-variable "set_decrypters_for" {
-  description = "Name of keys for which decrypters will be set."
-  type        = list(string)
-  default     = []
+variable "key_opts_default" {
```

Comment:
Collapsing many options into a single variable is an anti-pattern, we should continue to leave these as top-level variables.